bigbluebutton-config/bin/bbb-conf

```diff
@@ -204,6 +204,22 @@ if [ -f $SERVLET_DIR/WEB-INF/classes/bigbluebutton.properties ]; then
     fi
 fi
 
+get_url () {
+    api_method="getMeetings"
+    X=$( bbb-conf --secret | fgrep URL: )
```

Comment:
I don't like recursively running bbb-conf and grepping here - this is *already* in bbb-conf!

If there's not already a function in bbb-conf for getting the url and secret, consider refactoring the code that handles "bbb-conf --secret" to split that out into a function to share.